tests/test_core.py

```diff
@@ -28,6 +28,7 @@
     _compile_exact,
     _compile_fuzzy,
     _compile_nested,
+    compile,
```

Comment:
interestingly, `compile` seems to be a python global.

Reply:
:( is it easy to rename?

Reply:
not really needed, as `compile` is not a common used feature of python.